code/modules/research/designs/weapons.dm

```diff
@@ -212,9 +212,9 @@
 	id = "smg"
 	req_tech = list("combat" = 4, "materials" = 3)
 	build_type = PROTOLATHE
-	materials = list(MAT_IRON = 8000, MAT_SILVER = 2000, MAT_DIAMOND = 1000)
+	materials = list(MAT_IRON = 10000, MAT_GLASS = 5000)
```

Comment:
Understandable. I suppose I could up its cost further then without it being outright overpriced. Still,    I can imagine at least a few situations where bullet-firing weapons would be valuable to have around (e.g. wizard with Disable Tech, rechargers destroyed or unusable due to power failures, etc.)